hack/build-manifests.sh

```diff
@@ -55,7 +59,9 @@ function buildFlags {
 	--csv-version=${CSV_VERSION} \
 	--container-prefix=${CONTAINER_PREFIX} \
 	--replaces-version=${REPLACES_VERSION} \
-	--image-pull-policy=${IMAGE_PULL_POLICY}"
+	--image-pull-policy=${IMAGE_PULL_POLICY} \
+    --ims-conversion-container=${CONVERSION_CONTAINER} \
+    --ims-vmware-container=${VMWARE_CONTAINER}"
```

Comment:
spacing messed up. Tabs v Spaces?

Reply:
Yep. I use tab expansion in vim. FTL here. Should be fixed.